src/main/java/edu/kis/powp/jobs2d/command/CommandStatisticVisitor.java

```diff
@@ -0,0 +1,75 @@
+package edu.kis.powp.jobs2d.command;
+
+import java.util.Iterator;
+import java.util.Vector;
+
+public class CommandStatisticVisitor implements CommandVisitor {
+    private double totalLength = 0;
+    private Vector<Integer> previousCoordinates = null;
+    private Vector<Integer> currentCoordinates = null;
```

Comment:
Vector is not good abstraction here and null values could be avoided, e.g.:
https://docs.oracle.com/javase/8/docs/api/java/awt/Point.html